docs/root/configuration/http_filters/gzip_filter.rst

```diff
@@ -0,0 +1,47 @@
+.. _config_http_filters_gzip:
+
+Gzip
+====
+Gzip is an HTTP filter which enables Envoy to compress dispatched data
+from an upstream service upon client request. Compression is useful in
+situations where large payloads need to be transmitted without
+compromising the response time.
+
+Configuration
+-------------
+* :ref:`v1 API reference <config_http_filters_gzip_v1>`
+* :ref:`v2 API reference <envoy_api_msg_filter.http.Gzip>`
+
+.. attention::
+
+  Due to a known bug in the underlying Zlib library, window bits with value
+  eight does not work as expected. Therefore 9 is the smallest window size
```

Comment:
Specify explicitly that a value of 8 or smaller will be, for now, clamped to a minimum of 9?